assets/js/add-another-evidence.js

```diff
@@ -1,2 +1 @@
 new AddAnother($('.add-another-evidence'), '.remove-button-container')
-new AddAnother($('.add-another-input'), '.remove-button-container')
```

Comment:
moved this into a new file that more represents the page it serves, i.e details page